index.js

```diff
@@ -141,7 +133,7 @@ function validateTimestamp(requestBody) {
 
 
 // certificate validator express middleware for amazon echo
-var verifier = module.exports = function(cert_url, signature, requestBody, callback) {
+var verifier = module.exports = function(cert_url, signature, requestBody, callback, cache) {
```

Comment:
@martijnthe what is the use case for allowing a custom cache object to be passed in? 

I think for the sake of simplicity it might be better to just use `globalCache` and not expose this, unless there's a really compelling case for allowing this.